docs/content/Theming/06_browser_color.md

```diff
@@ -0,0 +1,46 @@
+@ngdoc content
+@name Browser Colors
+@description
+## This feature is **mobile** only
+
+![browser color](https://cloud.githubusercontent.com/assets/6004537/18006666/50519c7e-6ba9-11e6-905b-c3751c20549c.png)
+
+Enables browser header coloring with [Material Design Colors](https://material.google.com/style/color.html) and the theming system, <br/> 
+for more info please visit [here](https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color)
+ 
+## Usage
```

Comment:
I have a pet peeve about putting two headings right next to each other. Can you just thrown in a single sentence that says something like the following?

> Below are usage examples for both the Angular configuration phase, and during runtime.